api/tests/core/users/test_api.py

```diff
@@ -620,6 +625,73 @@ def test_with_set_postal_code(self):
         assert updated_user.departementCode == "75"
 
 
+class DepositWithIncidentsTest:
+    def test_deposit_bookings_with_incident(self):
+        deposit = users_factories.DepositGrantFactory()
+
+        bookings_factories.BookingFactory(deposit=deposit, amount=20)
+        bookings_factories.UsedBookingFactory(deposit=deposit, amount=45)
+
+        booking_reimbursed_2 = bookings_factories.ReimbursedBookingFactory(deposit=deposit, amount=40)
+        booking_reimbursed = bookings_factories.ReimbursedBookingFactory(deposit=deposit, amount=30)
+        incident = finance_factories.FinanceIncidentFactory()
+
+        finance_factories.IndividualBookingFinanceIncidentFactory(booking=booking_reimbursed, incident=incident)
+
+        # partial booking finance incident
+        finance_factories.IndividualBookingFinanceIncidentFactory(
+            booking=booking_reimbursed_2, incident=incident, newTotalAmount=3000
+        )
+
+        finance_api.validate_finance_incident(incident, False)
+
+        assert incident.status == finance_models.IncidentStatus.VALIDATED
+
+        # All bookings that are not cancelled
+        assert db.session.query(sa.func.get_deposit_balance(deposit.id, False)).first()[0] == 205
+        assert db.session.query(sa.func.get_deposit_balance(deposit.id, True)).first()[0] == 225
```

Comment:
Pourrais-tu mettre en commentaire le calcul permettant d'atteindre ces 2 montants ?

```suggestion
        assert db.session.query(sa.func.get_deposit_balance(deposit.id, False)).first()[0] == 205  # 300 - 20 - 45 - 30
        assert db.session.query(sa.func.get_deposit_balance(deposit.id, True)).first()[0] == 225  # 300 - 45 - 30
```

On peut même simplement écrire comme ça : 

```suggestion
        assert db.session.query(sa.func.get_deposit_balance(deposit.id, False)).first()[0] == 300 - 20 - 45 - 30
        assert db.session.query(sa.func.get_deposit_balance(deposit.id, True)).first()[0] == 300 - 45 - 30
```

Mais c'est peut-être une question de goût. :)